content/android/geolocation.md

```diff
@@ -0,0 +1,60 @@
+Skygear supports saving Geo Location as a field in record. Please be reminded
+that Geo Location is identified by latitude and longitude.
+
+### Saving a location in a record
+
+```java
+Location loc = /* get the location via LocationServices or create a new one */
+Record aRecord = new Record("Note");
+aRecord.set("place", loc);
+
+skygear.getPublicDatabase().save(aRecord, new RecordSaveResponseHandler(){
+    /* Implementation of response handler */
```

Comment:
Will it be too complicated to have a quick implementation showing the latitude of the saved record?